vehicle/vehicle_info_util/config/vehicle_info.param.yaml

```diff
@@ -9,3 +9,4 @@
     left_overhang: 0.1 # between left wheel center and vehicle left
     right_overhang: 0.1 # between right wheel center and vehicle right
     vehicle_height: 2.5
+    max_steer_angle: 0.70 # [rad]
```

Comment:
Could you apply pre-commit ?

Reply:
I'm wondering why it wasn't autofixed. :thinking:

Reply:
[e7b3df0](https://github.com/autowarefoundation/autoware.universe/pull/740/commits/e7b3df03471a6265b7aa99b74d70b3ab60498835)

Reply:
I did, manually.